katportalclient/client.py

```diff
@@ -1211,11 +1148,17 @@ def sensor_detail(self, sensor_name):
                         sensor_name,
                         [result['name'] for result in results[0:5]]))
         else:
-            raise tornado.gen.Return(results[0])
+            attrs = results[0].get('attributes')
+            result = { 'name': results[0].get('name'),
+                       'description': attrs.get('description'),
+                       'params': attrs.get('params'),
+                       'katcp_name': attrs.get('original_name'),
+                       'component': attrs.get('component')}
```

Comment:
OK, that is better, but what about the units?  We really need those too.

Also need similar changes around line 1143.